src/main/java/core/basesyntax/report/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.report;
+
+import core.basesyntax.operation.ShopService;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private ShopService shopService;

```

Comment:
Follow the encapsulation principle. You should hide the inner class elements with the help of access modifiers. It's a bad practice to make your class exposed. Here, the `shopService` field should be private.